packages/rocketchat-statistics/server/functions/get.js

```diff
@@ -32,10 +32,12 @@ RocketChat.statistics.get = function _getStatistics() {
 		}
 	});
 
-	if (statistics.wizard.allowMarketingEmails) {
-		const firstUser = RocketChat.models.Users.getOldest({ name: 1, emails: 1 });
-		statistics.wizard.contactName = firstUser && firstUser.name;
-		statistics.wizard.contactEmail = firstUser && firstUser.emails[0].address;
+	const firstUser = RocketChat.models.Users.getOldest({ name: 1, emails: 1 });
+	statistics.wizard.contactName = firstUser && firstUser.name;
+	statistics.wizard.contactEmail = firstUser && firstUser.emails && firstUser.emails[0].address;
+
+	if (RocketChat.models.Settings.findOne('Organization_Email')) {
```

Comment:
same as before, this query will always return the setting record, even if its value is blank, as the setting will always be there.. should this be testing its value i.e.: RocketChat.settings.get('Organization_Email') ?